src/primitives/block.cpp

```diff
@@ -207,12 +206,29 @@ bool CBlock::SignBlock(const CKeyStore& keystore)
             if (!keystore.GetKey(keyID, key))
                 return false;
 
-            //vector<unsigned char> vchSig;
             if (!key.Sign(GetHash(), vchBlockSig))
                  return false;
 
             return true;
         }
+        else if(whichType == TX_WITNESS_V0_KEYHASH)
```

Comment:
indentation incorrect

Reply:
I can't see the indention wrong. After I rewrote the indention to space (previously it was space too), git doesn't show any diff. Please check again.